sharding/indexHashedNodesCoordinator.go

```diff
@@ -134,7 +134,16 @@ func NewIndexHashedNodesCoordinator(arguments ArgNodesCoordinator) (*indexHashed
 		log.Error("saving initial nodes coordinator config failed",
 			"error", err.Error())
 	}
-
+	log.Info("new nodes config is set for epoch", "epoch", arguments.Epoch)
+	currentNodesConfig := ihgs.nodesConfig[arguments.Epoch]
+	if currentNodesConfig != nil {
```

Comment:
if no nodesConfig - shouldn't we return error ?

Reply:
fixed. Should not happen, because the nodesConfig[currentEpoch] was set just above